config/environments/production.rb

```diff
@@ -3,7 +3,7 @@
 	# Settings specified here will take precedence over those in config/application.rb
 
 	# Code is not reloaded between requests
-	config.cache_classes = true
+	config.cache_classes = false
```

Comment:
We need to cache_classes in production :)

Reply:
True. 🙂